devices/debian_wifi.py

```diff
@@ -1,88 +1,126 @@
-import re
+import re, time
 import debian
 import pexpect
+from countrycode import countrycode
+from lib.wifi import wifi_client_stub
+import pdb
 
-class DebianWifi(debian.DebianBox):
+class DebianWifi(debian.DebianBox, wifi_client_stub):
     '''Extension of Debian class with wifi functions'''
 
     model = ('debianwifi')
+    wifi_interface = "wlan0"
+    wifi_interface1 = "wlan1"
 
-    iface_wlan = "wlan1"
+    wifi_client_control = None
```

Comment:
We have stuck with iface_ all around why change it now?

Reply:
This is a wifi control to set wifi or wlan. I haven't removed the iface_ which we have been using